### DISCUSSION TITLE
How to enable the theme like 'Quick Start Guide' page

### DISCUSSION THREAD

**Discussion by thindo** [mmistakes/minimal-mistakes#2460] [Q&A] [answered]
Hello there,
I saw Quick-Start Guide page very useful, I want this theme for my jekyll, please help

Thanks,

**Comment by mmistakes** [comment, marked as answer]
Have a look at the source files for the demo site to see how I've built those pages.

https://github.com/mmistakes/minimal-mistakes/tree/master/docs
https://github.com/mmistakes/minimal-mistakes/blob/master/docs/_docs/01-quick-start-guide.md

**Comment by thindo** [comment]
![image](https://user-images.githubusercontent.com/39892337/77431949-04f63d80-6e10-11ea-90b0-33658a36097f.png)
I tried to edit url, but does not work, please see my github page https://thindo.github.io/

**Comment by iBug** [comment]
Perhaps you should get yourself familiar with Jekyll first? You at least need that page in the correct location before you can link to it.

**Comment by mmistakes** [comment]
As @iBug mentioned, you need to create the page to link to it. This is out of scope for the theme maintainers to support. It is assumed you have a working knowledge of Jekyll's core principles, how to create and setup [collections](https://jekyllrb.com/docs/collections/), work with `_data` files, etc.

Please consult their [official documentation](https://jekyllrb.com/docs/) first, then reach out in their [help forum](https://jekyllrb.com/docs/collections/).
